### PR TITLE
update iOSAppium2SetupGuide-README.md and added devices in caps file

### PR DESCRIPTION
**Pull request by LavanyaJK**
Added devices capability to ajio_browserstack_capabilities.json file
updated iOSAppium Readme file to work with Appium 2.0

### CODE REVIEW

File: caps/ajio_browserstack_capabilities.json [hunk 1]
```diff
@@ -50,7 +50,14 @@
           "cloud": {
             "cloudName": "browserstack",
             "url": "https://hub.browserstack.com",
-            "apiUrl": "https://api-cloud.browserstack.com/app-automate/"
+            "apiUrl": "https://api-cloud.browserstack.com/app-automate/",
+            "devices": [
+              {
```
Comment (anandbagmar) [inline caps/ajio_browserstack_capabilities.json:55]: in teswiz, we populate the devices at runtime. Why do we need to add this here?

File: docs/iOSAppium2SetupGuide-README.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Set the **usePrebuiltWDA** capability to false in a capabilities file, you can u
     
     "usePrebuiltWDA": false
 
+Add the capability **iosDeviceType** and set it to real in a capabilities file:
+
+    "iosDeviceType": real
```
Comment (anandbagmar) [inline docs/iOSAppium2SetupGuide-README.md:17]: I don't see this capability in the capabilities file. Can you update it there? Also, is this capability required for local "real" iOS device as well?

File: caps/ajio_browserstack_capabilities.json (hunk 1 above)
Comment (LavanyaJK) [reply caps/ajio_browserstack_capabilities.json]: as part of the examples, I added devices info,
have removed it.

File: docs/iOSAppium2SetupGuide-README.md (hunk 2 above)
Comment (LavanyaJK) [reply docs/iOSAppium2SetupGuide-README.md:17]: capabilities are in browserstack caps file, i have updated the sentence specifically for browserstack,
and we dont need this for local execution.

**Review by anandbagmar — APPROVED** [review]

**Pull request merged**